src/Workspaces/Core/Portable/PatternMatching/AllLowerCamelCaseMatcher.cs

```diff
@@ -20,25 +21,22 @@ internal partial class PatternMatcher
         /// a candidate using CamelCase matching. i.e. this code is responsible for finding the
         /// match between "cofipro" and "CodeFixProvider". 
         /// </summary>
-        private struct AllLowerCamelCaseMatcher
+        private ref struct AllLowerCamelCaseMatcher
         {
             private readonly bool _includeMatchedSpans;
             private readonly string _candidate;
-            private readonly ArrayBuilder<TextSpan> _candidateHumps;
-            private readonly TextChunk _patternChunk;
             private readonly string _patternText;
```

Comment:
💭 Not a huge fan of conceptually separating `_patternText` from `_patternChunk`. As a follow-up, I would be interested to see how much complexity is added by changing remaining uses of `_patternChunk` to take a `TextChunk` parameter matching all the new uses of what used to be `_patternChunk`.

Reply:
well.. tehre are no other uses of _patternChunk.  We were only passing it in to get the Text of it.  So removing that was fine.